controllers/providers/assistant/gslb.go

```diff
@@ -221,25 +221,50 @@ func (r *GslbLoggerAssistant) InspectTXTThreshold(fqdn string, edgeDNSServerPort
 	return errors.NewResourceExpired(fmt.Sprintf("Can't find split brain TXT record at EdgeDNS server(%s) and record %s ", ns, fqdn))
 }
 
+func GetARecords(msg *dns.Msg) []string {
+	var ARecords []string
+	for _, nsA := range msg.Answer {
+		IP := strings.Split(nsA.String(), "\t")[4]
+		ARecords = append(ARecords, IP)
```

Comment:
consider this:
```suggestion
		ip := nsA.(*dns.A).A.String()
		ARecords = append(ARecords, ip)
```

Reply:
wasn't aware it is available this way, will try it out 👍